### ISSUE TITLE
Website password protection should be clearer

### ISSUE THREAD

**Issue by shepster** [OWASP/passfault#37]
On the Passfault website (https://passfault.appspot.com) the password protection lists various encryption schemes. I think the average user has no idea what those mean. It would be more helpful to list operating systems rather hash functions. For example, if I'm on Windows 7 know strong is my password in that environment? Perhaps after the has algorithm it could list the OSs that use that algorithm, or if there are too many options for a popup have a web page with a table.

**Comment by slambert** [comment]
I agree. Or at least some cursory explanation on the site on how most passwords are protected in typical environments: email, social networks, your OS, etc. 

Say I'm some random person and wanted to make sure I had a strong password for facebook? Which encryption should I choose to know if my password is strong enough? I guess defaulting the weakest encryption is safest, and Microsoft Windows NT LAN Manager seems to be the weakest, but some clarity would be helpful. I feel like I should know, and I don't.
